notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2211 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        machine_type='n1-highcpu-4', 
</pre><p>Use a constant for this too for consistency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
Sure. Added.